x/auth/ante/ante.go

```diff
@@ -18,7 +18,7 @@ type HandlerOptions struct {
 	BankKeeper             types.BankKeeper
 	ExtensionOptionChecker ExtensionOptionChecker
 	FeegrantKeeper         FeegrantKeeper
-	SignModeHandler        authsigning.SignModeHandler
+	SignModeHandler        *txsigning.HandlerMap
```

Comment:
This is the core change driving this PR.

Reply:
we should get a changelog & upgrade entry for this as some users may not use handler options and copy the contents